src/api/models/vn/length.ts

```
@@ -0,0 +1,24 @@
+export const VN_LENGTHS = ['veryshort', 'short', 'medium', 'long', 'verylong'] as const;
+
+export type VnLength = typeof VN_LENGTHS[number];
+
+const LENGTH_READABLE_MAP: Readonly<Record<VnLength, string>> = {
+  veryshort: 'Very short',
+  short: 'Short',
+  medium: 'Medium',
+  long: 'Long',
+  verylong: 'Very Long',
+};
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export namespace VnLength {
+
+  /**
+   * Converts length to a readable equivalent.
+   * @param value Vn length.
+   */
+  export function toReadable(value: VnLength | null): string {
+    const UNKNOWN_LENGTH = 'Unknown';
+    return value !== null ? LENGTH_READABLE_MAP[value] ?? UNKNOWN_LENGTH : UNKNOWN_LENGTH;
```

We could extract `"Unknown"` into global constant.

---

There is no other `Unknow` for now, each model has it's own "default"

---

@Moltenship It's true, yet all of them use "Unknown", we could use it as default unknown value for all models.

---

I don't think we need to work with `Unknown`, better to throw an error right away